jdaviz/configs/imviz/plugins/links_control/links_control.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <v-card flat tile>
+    <v-card>
+      <v-card-text>
+        <v-container>
+          <v-row>
+            <v-col>
+              <v-select
+                :items="link_types"
+                @change="link_type_selected"
+                label="Link type"
+                hint="Type of linking to be done."
+                persistent-hint
+              ></v-select>
+            </v-col>
+          </v-row>
+        </v-container>
+        <v-container>
+          <v-row>
+            <v-col>
+              <v-select
+                :items="wcs_fallback_schemes"
+                @change="wcs_fallback_scheme_selected"
+                label="WCS fallback scheme"
+                hint="If WCS linking fails, fall back to linking by pixels or not at all."
+                persistent-hint
+              ></v-select>
+            </v-col>
+          </v-row>
+        </v-container>
+        <v-container>
+          <v-row>
+            <v-col>
+              <v-select
+                :items="wcs_use_affine_options"
+                @change="affine_option_selected"
+                label="Use affine transform"
+                hint="Use affine transform for WCS, if possible."

```

Comment:
Suggested wording:

```suggestion
                label="Use fast approximation if possible"
                hint="Use fast approximation for image alignment if possible (accurate to <1 pixel)"

```

The use of 'affine' might confuse people into thinking we are ignoring projections/distortions when we are not - two images with identical heavily distorted WCSes can have their *difference* exactly represented by an affine transform. The fact the approximation is affine is not really important, what is important is the accuracy.

Reply:
Sounds reasonable, but I'll wait till Kyle finalizes the UI design first. 😸

Reply:
OK. Done.